blog/_posts/2022-02-24-rancher-desktop-container-runtimes.md

```diff
@@ -0,0 +1,118 @@
+---
+slug: "rancher-desktop-container-runtimes"
+date: 2022-02-24
+author: milas
+layout: blog
+title: "Rancher Desktop: Should You Use containerd Or dockerd?"
+image: "/assets/images/rancher-desktop-container-runtimes/title.jpg"
+image_caption: 'Photo by <a rel="noopener noreferrer" href="https://unsplash.com/photos/u0vgcIOQG08">Jens Lelie</a> on Unsplash'
+description: "Configure Tilt to build images with kim for use with Rancher Desktop"
+tags:
+- docker
+- moby
+- containerd
+- rancher
+- kubernetes
+---
+
+[Rancher Desktop][rancher-desktop], a lightweight and local Kubernetes cluster solution, has been evolving _fast_ over the past few months!
+
+Early releases used [containerd][] behind-the-scenes with an experimental CLI tool, [`kim`][kim], to build images.
+I wrote up a blog post about how easy it was to integrate with Tilt: [Writing Yet Another Custom Image Builder][blog-kim-extension].
+
+Since then, however, Rancher Desktop has started also supporting Docker (aka `dockerd (moby)`) as an alternative to containerd.
+Furthermore, [`kim`][kim] is no longer under active development.
+
+While this might seem like a lot of churn, none of this is necessarily a bad thing!
+In fact, when I was first writing about all this, our CTO, Nick, even predicted some of these changes.
```

Comment:
Rather than use this as a time for gloating, I would try to focus more on the user benefits (which is what's driving these changes)

`nerdctl` is a project that uses a lot of the same architecture + components as `kim`. By focusing all the community energy on a single project, we'll get a better, more well-maintained tool overall, and both `kim` and `nerdctl` users will benefit.

Reply:
Oof, yeah definitely didn't mean for it to come off as gloating!

I like your description - better captures what I was trying to state, which is basically that it's not surprising for an early, "sandbox" project to have some big changes